src/proto/mgmt/mgmt.proto

```diff
@@ -33,7 +33,6 @@ import "acl.proto"; // ACL-related requests
 // Used in gRPC fanout to operate on hosts with multiple ranks.
 message RanksReq {
 	repeated uint32 ranks = 1;
-	float timeout = 2;
```

Comment:
If a field is going away, might want to mark it reserved so the number doesn't get reused. Same API concern as above - not sure how much we're interested in preserving compatibility.

Reply:
yes , we need a definitive decision on this

Reply:
I suggest we start reserving fields after 1.0, this PR is targeted for 1.0